src/internet_identity/src/storage/tests.rs

```diff
@@ -101,7 +101,7 @@ fn should_read_previous_write() {
 
 #[test]
 fn should_serialize_first_record() {
-    const EXPECTED_LENGTH: usize = 253;
+    const EXPECTED_LENGTH: usize = 261;
```

Comment:
Not for this PR but you could get this number from encoding the anchor instead of through magic.